test/integration/lockfile/test_ci_overrides.py

```diff
@@ -13,7 +13,7 @@ def test_graph_build_order_override_error():
               |--------------/
       |-----override------> zlib/1.3
     """
-    c = TestClient()
+    c = TestClient(light=True)
```

Comment:
Settings copy was a prevalent time sink as per cProfile output of this file, and in my Parallels machine this goes from 63s to 24